rasa/core/actions/action.py

```diff
@@ -591,7 +591,103 @@ def action_response_format_spec() -> Dict[Text, Any]:
+                                                "items": {
+                                                    "type": "object",
+                                                    "properties": {
+                                                        "start": {"type": "integer"},
+                                                        "end": {"type": "integer"},
+                                                        "name": {"type": "string"},
+                                                        "confidence": {
+                                                            "type": "number"
+                                                        },
+                                                        "extractor": {"type": "string"},
+                                                        "value": {"type": "object"},
+                                                    },
+                                                },
+                                            },
+                                        },
+                                    },
+                                }
+                            },
+                            {
+                                "properties": {
+                                    "event": {"const": "action"},
+                                    "policy": {},
+                                    "confidence": {},
+                                    "name": {"type": "string"},
+                                }
+                            },
+                            {
+                                "properties": {
+                                    "event": {"const": "slot"},
+                                    "name": {"type": "string"},
+                                    "value": {},
+                                }
+                            },
+                            {
+                                "properties": {
+                                    "event": {"const": "entities"},
+                                    "entities": {
+                                        "type": "array",
+                                        "items": {
+                                            "type": "object",
+                                            "properties": {
+                                                "start": {"type": "integer"},
+                                                "end": {"type": "integer"},
+                                                "name": {"type": "string"},
+                                                "confidence": {"type": "number"},
+                                                "extractor": {"type": "string"},
+                                                "value": {"type": "object"},
+                                            },
+                                        },
+                                    },
+                                }
+                            },
```

Comment:
what about deduplicating this as you use the same for `UserUttered`?

Reply:
By deduplicating you mean extracting this dictionary in a variable and assigning the variable instead?